index.html

```diff
@@ -38,6 +38,128 @@ <h5>LET’S CONNECT</h5>
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+        <div class="grid-item show-case2">
+          <div class="content">
+            <h4>Multi-Post Stories</h4>
+            <ul class="flex">
+              <li class="ttt">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+        <div class="grid-item show-case3">
+          <div class="content">
+            <h4>Tonic</h4>
+            <ul class="flex">
+              <li class="ttt">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+        <div class="grid-item show-case4">
+          <div class="content">
+            <h4>Multi-Post Stories</h4>
+            <ul class="flex">
+              <li class="ttt">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+      </section>
+      <section id="about-me" class="flex">
+        <h2>About Myself</h2>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h5>LET’S CONNECT</h5>
+        <ul class="flex">
+          <li><img src="img/Icons/Twitter.png" alt="Twitter Icon" /></li>
+          <li>
+            <img src="img/Icons/Linkedin-icon.png" alt="Linkedin-icon" />
+          </li>
+          <li><img src="img/Icons/M.png" alt="M icon" /></li>
+          <li><img src="img/Icons/Github.png" alt="Github Icon" /></li>
+          <li><img src="img/Icons/hands.png" alt="hands" /></li>
```

Comment:
Each icon needs to have an anchor tag in order to lead the user to your media accounts;

- [ ] Kindly, wrap the icons inside anchor tags for the reason stated above; :+1: